From 28f6a9e667c94c2eba4b79c61f77953ff140df0a Mon Sep 17 00:00:00 2001
From: Du Chenyang <chenyang.du@intel.com>
Date: Sat, 8 Jun 2013 11:25:10 +0800
Subject: Replace inline with __inline__ in macro __BIONIC_FORTIFY_INLINE

Bug: 114804

umask multi-define error will occur in external/pcsc when try to
enable fortify. Replace inline with __inline__ to fix it from bionic.

Category: Device enablement
Domain: AOSP-bionic
Source: Internal
Upstream-candidate: Yes

Change-Id: I428babc9dd87d7ad424268e7e3120e2a075b9574
Signed-off-by: Du Chenyang <chenyang.du@intel.com>
---
 libc/include/sys/cdefs.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libc/include/sys/cdefs.h b/libc/include/sys/cdefs.h
index a4c1aff..2a1396e 100644
--- a/libc/include/sys/cdefs.h
+++ b/libc/include/sys/cdefs.h
@@ -535,7 +535,7 @@
 #endif
 
 #define __BIONIC_FORTIFY_INLINE \
-    extern inline \
+    extern __inline__ \
     __attribute__ ((always_inline)) \
     __attribute__ ((gnu_inline))
 #endif
-- 
1.7.4.1

